docs/minio-admin-complete-guide.md

```diff
@@ -454,3 +456,25 @@ mc admin heal -r myminio/mybucket
 ```sh
 mc admin heal -r myminio/mybucket/myobjectprefix
 ```
+
+<a name="top"></a>
+### Command `top` - provide top like statistics for Minio
+NOTE: This command is only applicable for distributed Minio setup.
```

Comment:
Do we want to be pedantic in this description "...for distributed MinIO setup, i.e., not for single node or gateway deployments" ?  Might help clear up some confusion

Reply:
done